Sources/Models/MessageStyle.swift

```diff
@@ -28,12 +28,12 @@ public enum MessageStyle {
 
     // MARK: - TailCorner
 
-    public enum TailCorner {
+    public enum TailCorner: String {
 
-        case topLeft
-        case bottomLeft
-        case topRight
-        case bottomRight
+        case topLeft = "topLeft"
```

Comment:
FYI this is not necessary. If you don't provide an explicit rawValue for `String` it will default to using the case name as a `String`

Reply:
👍